go.mod

```diff
@@ -4,7 +4,7 @@ go 1.16
 
 require (
 	cloud.google.com/go/compute v1.7.0
-	github.com/Microsoft/go-winio v0.4.15
```

Comment:
This change seems to be breaking the build.

Reply:
> This change seems to be breaking the build.

The main change is about the use of `errors.Is`. If we do not use pkg/errors, we have to alter it to use strings.Contains when checking if an error is wrapped.